src/get-data.ts

```diff
@@ -19,7 +17,7 @@ const getData = async (): Promise<PRData> => {
       functions: { hit: 0, found: 0, percent: 0, diff: 0 },
       branches: { hit: 0, found: 0, percent: 0, diff: 0 },
     },
-    token: authToken,
+    token: authToken ?? null,
```

Comment:
We should avoid explicitly using `null` and just make this `undefined` if it's missing

Reply:
I made it null because undefined can't be serialized as json and we include the token in the post request to the endpoint. alternatively if the token doesn't exist I can remove the field from the post request - this would also then require changing one line of code on the service. Let me know your preference.

Reply:
@iansu

Reply:
We just shouldn't send a request at all if the token is not provided